sqlserver/datadog_checks/sqlserver/connection.py

```diff
@@ -39,7 +40,7 @@ def split_sqlserver_host_port(host):
     """
     if not host:
         return host, None
-    host_split = [s.strip() for s in host.split(',')]
+    host_split = [s.strip() for s in host.split(",")]
```

Comment:
Can we undo all unrelated formatting changes (string single quotes to double quotes, and wrapping lines more tightly than the built-in style formatter)? It's making the diff larger than it needs to be.

Reply:
the line-wrapping changes are required by CI - without them, I get CI failures. 
I can try to undo the quote changes though.

Reply:
Removed all the quote changes, and all the formatting changes are in a single commit at the end so it should be easy to review without them.

Reply:
Thanks, looks good. I'm curious how those longer lines that got wrapped were passing CI before.

Reply:
Possibly landed before this became a requirement? I'm formatting the whole file to format my own changes, so totally possible I'm catching stuff that hasn't been touched in a while.